media-gfx/blender/blender-9999.ebuild

```diff
@@ -182,10 +196,13 @@ src_unpack() {
 
 		git-r3_fetch "${ADDONS_EGIT_REPO_URI}"
 		git-r3_checkout "${ADDONS_EGIT_REPO_URI}" "${S}/scripts/addons"
-		# TODO
-		#if use test; then
 
-		#fi
+		if use test; then
```

Comment:
Why did you add these svn commands back?

I think they will fail regardless because we do not inherit the svn eclass anymore.

Because upstream migrated to git-lfs, we should not use subversion anymore as those repos are not updated anymore.

To get the tests to work I think we only need a:
```
 if use test; then
 EGIT_SUBMODULES=( '*' '-lib/*' )
 fi
 ```
 at the `inherit git-r3` to get the test repo checkedout.

Reply:
I knew there was a reason that was commented out. Will fix in a bit.

Reply:
Note that this will only work if you have the changes that was merged into the main repo 2 days ago:
https://github.com/gentoo/gentoo/pull/35911

Reply:
Right, I we still have to download the test files and host them ourselves if we download them as a tar.
In the past this has proven quite difficult as I don't have a place to host these myself.

Perhaps we could simply use git+git-lfs to fetch for test files for non live ebuilds as well? :thinking: 

I have additional remarks that I think might be better by me doing a PR to your branch. I'll try to have something later today.

Reply:
https://github.com/gentoo/gentoo/blob/master/eclass/git-r3.eclass#L36 this makes it impossible to use git-r3 eclass to do that. We could call git ourselves. But I'm not really sure if that the right thing to do. I think filing a bug with blender to enable the setting to make archives including the actual files might be better.

Reply:
gitea doesn't support packaging the LFS files into the repo .zip ATM.
I've also talked with the Blender admins in charge of this (I'm a Blender Employee as well), and sadly they are not too convinced that we should create zip files with the test files for the releases.
They are mainly concerned about the additional storage and bandwidth requirements as we currently are struggling a bit with that.

So I guess we would have to still host the test files ourselves like sam tried to do for a few releases. :(

Reply:
Or hmm...
@mgorny how do you feel about being able to not have the "live" tag for git checkouts that is using tags?
This way we would be able to download releases from repos with lfs files or submodules without having to package it ourselves.

Reply:
I suspect the answer is going to be "it is inconvenient for a reason, because people aren't supposed to use it just for the sake of convenience".

The point of the "live" property is more than just "we assume the exact commit version is unknown".